view/src/worker.rs

```diff
@@ -33,7 +36,8 @@ pub struct Worker {
     fvk: FullViewingKey, // TODO: notifications (see TODOs on ViewService)
     error_slot: Arc<Mutex<Option<anyhow::Error>>>,
     sync_height_tx: watch::Sender<u64>,
-    tm_client: tendermint_rpc::HttpClient,
+    tm_client: TendermintServiceClient<Channel>,
+    tm_proxy_client: TendermintProxyServiceClient<Channel>,
```

Comment:
Could be later, but rather than maintaining a bunch of different clients, maybe we should just maintain one `Channel`, then whenever we need an XYZClient, we would clone the `Channel` and build the client we needed? https://docs.rs/tonic/latest/tonic/transport/struct.Channel.html#multiplexing-requests